lib/classes/Utils.js

```diff
@@ -329,6 +329,7 @@ class Utils {
             timestamp: (new Date()).getTime(),
             timezone: (new Date()).toString().match(/([A-Z]+[+-][0-9]+)/)[1],
             operatingSystem: process.platform,
+            agent: (process.env.SERVERLESS_DASHBOARD) ? 'dashboard' : 'cli',
```

Comment:
`agent` or `userAgent` (`userAgent` is more common to me as browser trackings also use it).